contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -1036,11 +1035,12 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
     }
 
     /// @notice Returns a counter that MUST change it's value when any of the following happens:
-    ///     1. a node operator's key(s) is added
-    ///     2. a node operator's key(s) is removed
-    ///     3. a node operator's ready to deposit keys count is changed
+    ///     1. a node operator's validator(s) is added
+    ///     2. a node operator's validator(s) is removed
+    ///     3. a node operator's ready to deposit validators count is changed
     ///     4. a node operator was activated/deactivated
-    function getValidatorsKeysNonce() external view returns (uint256) {
+    ///     5. a node operator's validator(s) is used for the deposit
+    function getDepositsDataNonce() external view returns (uint256) {
```

Comment:
the commonly used term is "deposit data", in the singular form

Reply:
I'd rename to `getDepositNonce()`

Reply:
Renamed to `getDepositNonce()`